airflow/models/abstractoperator.py

```diff
@@ -44,7 +44,7 @@
 if TYPE_CHECKING:
     from collections.abc import Mapping
 
-    import jinja2  # Slow import.
+    import jinja2  # Slow imports.
```

Comment:
I'd propose to make this to another PR. Seems unrelated.

Reply:
Yeah. I could not help myself.